Students/tylerkobienia/BLINK_LED/BLINK_LED.ino

```diff
@@ -0,0 +1,11 @@
+void setup() {
+  pinMode(LED_BUILTIN, OUTPUT);
+}
+
+
+void loop() {
+  digitalWrite(LED_BUILTIN, HIGH);  
+  delay(500);                     
+  digitalWrite(LED_BUILTIN, LOW);   
+  delay(500);                      
+}
```

Comment:
Please review the three items to do in Part 1. There should be a slow flash that happens just once, then quicker flash that repeats. Also, comments!